packages/react-components/react-tag-picker-preview/src/components/TagPickerControl/useTagPickerControlStyles.styles.ts

```diff
@@ -156,8 +156,8 @@ export const useTagPickerControlStyles_unstable = (state: TagPickerControlState)
     styles.root,
     styles[state.size],
     styles[state.appearance],
-    !state.disabled && state.appearance === 'outline' && styles.outlineInteractive,
-    state.disabled && styles.disabled,
+    // !state.disabled && state.appearance === 'outline' && styles.outlineInteractive,
+    // state.disabled && styles.disabled,
```

Comment:
should these stay commented?

Reply:
I'll keep this commented for now as I still haven't investigate all the cases of disabled yet. There are too many controls involved in this component, disabling might not be trivial.